code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -290,21 +290,15 @@
 	occupant_typecache = typecacheof(/mob/living/carbon)
 
 /obj/machinery/bci_implanter/on_deconstruction()
-	var/obj/item/organ/internal/cyberimp/bci/bci_to_implant_resolved = bci_to_implant?.resolve()
-	bci_to_implant_resolved?.forceMove(drop_location())
-	bci_to_implant = null
+	drop_stored_bci()
 
 /obj/machinery/bci_implanter/Destroy()
-	QDEL_NULL(bci_to_implant)
+	drop_stored_bci()
```

Comment:
Destroy() should never add things to the world. bci_implanter/Destroy() should destroy the BCI.

Reply:
Got it.